Campaigns/Ransomware hits healthcare - Alternate Data Streams use.txt

```diff
@@ -0,0 +1,10 @@
+//Find use of Alternate Data Streams (ADS) for anti-forensic purposes.
+// Alternate Data Streams execution 
+DeviceProcessEvents 
+| where Timestamp > ago(7d) 
+// Command lines used 
+| where ProcessCommandLine startswith "-q -s" and ProcessCommandLine hasprefix "-p" 

```

Comment:
maybe use has here instead of hasprefix? hasprefix is correct but it might be a bit confusing for someone who doesn't know the language well - sounds a bit similar to startswith